playbook/app/controllers/playbook/application_controller.rb

```diff
@@ -6,7 +6,6 @@ class ApplicationController < ActionController::Base
     helper Playbook::ApplicationHelper
     helper Playbook::PbDocHelper
     helper Playbook::PbSampleHelper
-    append_view_path Playbook::Engine.root + "app/pb_kits"
```

Comment:
Does this change our Getting started docs? https://playbook.powerapp.cloud/getting_started

Reply:
It does, I'll update it

Reply:
Actually, no. Our `getting_started` has no mention to the view paths. But it should not be necessary because even today, `Playbook::Engine` configures the view paths already.